src/main/java/online/shop/controller/admin/GetAllOrdersAdminController.java

```diff
@@ -0,0 +1,25 @@
+package online.shop.controller.admin;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import online.shop.lib.Injector;
+import online.shop.model.Order;
+import online.shop.service.OrderService;
+
+public class GetAllOrdersAdminController extends HttpServlet {
```

Comment:
```suggestion
public class GetAllOrdersController extends HttpServlet {
```

you should also create GetUserOrdersController and its JSP